src/Http/ServerRequest.php

```diff
@@ -2324,23 +2327,28 @@ public function withRequestTarget($target)
      * application relative path without base directory, and the query string
      * defined in the SERVER environment.
      *
+     * @param bool $base Include the base path, set to false to trim the base path off.
      * @return string
      */
-    public function getRequestTarget()
+    public function getRequestTarget($base = false)
```

Comment:
`getRequestTarget()` is an interface method declared in `Psr\Http\Message\ServerRequestInterface`, you can't change it's signature.

Reply:
Ah no wonder there's a ton of failures, I missed that bit.